.circleci/config.yml

```diff
@@ -25,12 +25,7 @@ jobs:
       - restore_cache:
           keys:
             - go-getter-modcache-v1-{{ checksum "go.mod" }}
-
-      # Save go module cache if the go.mod file has changed
-      - save_cache:
-          key: go-getter-modcache-v1-{{ checksum "go.mod" }}
-          paths:
-            - "/go/pkg/mod"
+            - go-getter-modcache-v1
```

Comment:
Curious what this for here. This would never match since the caches are saved with `go-getter-modcache-v1-{{ checksum "go.mod" }}` so the key would need to be `go-getter-modcache-v1-` if you meant to match when there is no `go.mod` file to checksum.

Reply:
You probably don't need this at all.

Reply:
I added it per a comment Aaron made, but I'm happy enough to take it out as well... Aaron's suggestion was that it would allow you to get a previous cache if one existed but the checksum didn't match. My assumption there was that you pick up a little efficiency because when it gets to running `go mod` as part of the tests it'll only have to grab packages that aren't already in the saved cache.

I could go either way on it since I don't think the savings is that great and looking at it in the YAML is actually a smidge confusing if you don't know that circle allows the fallback there.

...nevermind, just talked myself around to taking it out based on that. :D